Program type already present

When I add your dependency I get an error during the build
'Program type already present: com.google.gson.DefaultDateTypeAdapter'
'Program type already present: com.google.gson.ExclusionStrategy'

Could you please help me solve this issue?

Here is the log:
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Execution failed for task ':app:transformClassesAndResourcesWithProguardForDebug'.
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Job failed, see logs for details
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]  Run with --scan to get full insights.
18:40:53.509 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
18:40:53.510 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Exception is:
18:40:53.510 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:transformClassesAndResourcesWithProguardForDebug'.
18:40:53.510 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:95)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:91)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ResolveBuildCacheKeyExecuter.execute(ResolveBuildCacheKeyExecuter.java:79)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:57)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:119)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ResolvePreviousStateExecuter.execute(ResolvePreviousStateExecuter.java:43)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:93)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:45)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:94)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:56)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:55)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:67)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:49)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:315)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$CallableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:305)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:101)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.call(DelegatingBuildOperationExecutor.java:36)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:49)
18:40:53.511 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:43)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:355)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:343)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:336)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:322)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:134)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker$1.execute(DefaultPlanExecutor.java:129)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:202)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:193)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:129)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: java.lang.RuntimeException: Job failed, see logs for details
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.ProGuardTransform.transform(ProGuardTransform.java:209)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:239)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.pipeline.TransformTask$2.call(TransformTask.java:235)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.builder.profile.ThreadRecorder.record(ThreadRecorder.java:102)
18:40:53.512 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.pipeline.TransformTask.transform(TransformTask.java:230)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.project.taskfactory.IncrementalTaskAction.doExecute(IncrementalTaskAction.java:47)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:41)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:28)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$2.run(ExecuteActionsTaskExecuter.java:284)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:301)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:293)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:175)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:91)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:273)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:258)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.access$200(ExecuteActionsTaskExecuter.java:67)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$TaskExecution.execute(ExecuteActionsTaskExecuter.java:145)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.ExecuteStep.execute(ExecuteStep.java:49)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CancelExecutionStep.execute(CancelExecutionStep.java:34)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.TimeoutStep.executeWithoutTimeout(TimeoutStep.java:69)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.TimeoutStep.execute(TimeoutStep.java:49)
18:40:53.513 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CatchExceptionStep.execute(CatchExceptionStep.java:33)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CreateOutputsStep.execute(CreateOutputsStep.java:50)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SnapshotOutputStep.execute(SnapshotOutputStep.java:43)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SnapshotOutputStep.execute(SnapshotOutputStep.java:29)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CacheStep.executeWithoutCache(CacheStep.java:134)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CacheStep.lambda$execute$3(CacheStep.java:83)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CacheStep.execute(CacheStep.java:82)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.CacheStep.execute(CacheStep.java:36)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.PrepareCachingStep.execute(PrepareCachingStep.java:33)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.StoreSnapshotsStep.execute(StoreSnapshotsStep.java:38)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.StoreSnapshotsStep.execute(StoreSnapshotsStep.java:23)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SkipUpToDateStep.executeBecause(SkipUpToDateStep.java:96)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SkipUpToDateStep.lambda$execute$0(SkipUpToDateStep.java:89)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:52)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.steps.SkipUpToDateStep.execute(SkipUpToDateStep.java:36)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.internal.execution.impl.DefaultWorkExecutor.execute(DefaultWorkExecutor.java:34)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:91)
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	... 33 more
18:40:53.514 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: java.io.IOException: java.lang.NullPointerException
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.ProGuardTransform.doMinification(ProGuardTransform.java:280)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.ProGuardTransform.access$000(ProGuardTransform.java:67)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.ProGuardTransform$1.run(ProGuardTransform.java:178)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.builder.tasks.Job.runTask(Job.java:47)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.tasks.SimpleWorkQueue$EmptyThreadContext.runTask(SimpleWorkQueue.java:41)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.builder.tasks.WorkQueue.run(WorkQueue.java:282)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Caused by: java.lang.NullPointerException
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.ProgramClass.constantPoolEntryAccept(ProgramClass.java:537)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.UsageMarker.markConstant(UsageMarker.java:1246)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.UsageMarker.visitRequiresInfo(UsageMarker.java:1040)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.attribute.module.ModuleAttribute.requiresAccept(ModuleAttribute.java:138)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.UsageMarker.visitModuleAttribute(UsageMarker.java:739)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.attribute.module.ModuleAttribute.accept(ModuleAttribute.java:99)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.ProgramClass.attributesAccept(ProgramClass.java:619)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.UsageMarker.markProgramClassBody(UsageMarker.java:124)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.UsageMarker.visitProgramClass(UsageMarker.java:94)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.visitor.MultiClassVisitor.visitProgramClass(MultiClassVisitor.java:67)
18:40:53.515 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.visitor.MultiClassVisitor.visitProgramClass(MultiClassVisitor.java:67)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.visitor.ClassNameFilter.visitProgramClass(ClassNameFilter.java:128)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.ProgramClass.accept(ProgramClass.java:430)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.ClassPool.classesAccept(ClassPool.java:124)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.visitor.AllClassVisitor.visitClassPool(AllClassVisitor.java:45)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.visitor.MultiClassPoolVisitor.visitClassPool(MultiClassPoolVisitor.java:85)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.classfile.ClassPool.accept(ClassPool.java:110)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.shrink.Shrinker.execute(Shrinker.java:90)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.ProGuard.shrink(ProGuard.java:381)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at proguard.ProGuard.execute(ProGuard.java:145)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.BaseProguardAction.runProguard(BaseProguardAction.java:66)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	at com.android.build.gradle.internal.transforms.ProGuardTransform.doMinification(ProGuardTransform.java:274)
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 	... 5 more
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
18:40:53.516 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]

--- a/certificatetransparency-android/consumer-proguard-rules.pro
+++ b/certificatetransparency-android/consumer-proguard-rules.pro
@@ -80,3 +80,13 @@
 
 # Ensure chain cleaner classes are kept as they're loaded through reflection
 -keep class com.babylon.certificatetransparency.chaincleaner.* { *; }
+
+
+# Specifically for ProGuard (not needed for R8)
+-dontwarn module-info
+-dontnote module-info
+-dontwarn kotlinx.coroutines.debug.AgentPremain
+-dontwarn kotlinx.coroutines.debug.AgentPremain$DebugProbesTransformer
+-dontwarn org.bouncycastle.jce.provider.OcspCache
+-dontwarn org.bouncycastle.jce.provider.ProvOcspRevocationChecker
+-dontwarn org.bouncycastle.jsse.util.CustomSSLSocketFactory

